tester/src/test/java/com/graphicsfuzz/tester/ReducerUnitTest.java

```diff
@@ -101,6 +105,11 @@ public void testReductionSteps() throws Exception {
   private void testGenerateAndReduce(File originalShaderJobFile,
                                      List<ITransformationSupplier> transformations,
                                      RandomWrapper generator) throws Exception {
+
+    // TODO: We reduced this due to slow shader run time. Consider reverting to 100000 when run
+    //  time is improved.
```

Comment:
Could you file an issue for this and change the TODO to TODO(issue number)?

I would like for us to follow that practice so that we can track the TODOs.

Reply:
Done